include/rcppmath/rolling_mean_accumulator.hpp

```diff
@@ -27,20 +27,30 @@ namespace rcppmath
 
 /**
  * \brief Simplification of boost::accumulators::accumulator_set<double,
- *  bacc::stats<bacc::tag::rolling_mean>> to avoid dragging boost dependencies
+ *  bacc::stats<bacc::tag::rolling_mean>> to avoid dragging boost dependencies.
```

Comment:
I know you didn't add this, but I'd honestly just remove this sentence.  This is an implementation detail, and doesn't tell us what the class is meant to do.

If we really want to keep it, then I'd lead with "Computes the mean of the last accumulated elements", and then somewhere in the extended description mention that it is a simplified version of boost rolling_mean.

Reply:
I concur that this line should be changed. I'll remove the lengthy signature and mention boost rolling_mean in the extended description then.

Reply:
Fixed in 6cd9a07.